### DISCUSSION TITLE
Question about language support

### DISCUSSION THREAD

**Discussion by sundevista** [khoj-ai/khoj#169] [Q&A] [answered]
Hi! Does Khoj support any language or English only? If it's only English, can I somehow train it for other languages?

**Comment by debanjum** [comment, marked as answer]
By default Khoj only works with English but you can [configure it to use a multi-lingual model](https://github.com/debanjum/khoj#search-across-different-languages). 
This should then allow you to search for notes in 50+ languages[^1]

[^1]: https://www.sbert.net/docs/pretrained_models.html#multi-lingual-models

**Reply by sundevista** [reply]
Thank you!
